src/screens/layout.less

```diff
@@ -45,7 +46,7 @@
   overflow:hidden;
 	overflow-y: auto;
   bottom:0px;
-  border-right:1px solid #CFD8DC;
```

Comment:
This color is intentional, to draw contrast between the left and center panel, which can have unrelated content. Maybe we can create some sort of special variable for this? `@splitter-border-color: #CFD8DC` or something like that?